src/gpu/nvidia/cudnn_convolution_impl.hpp

```diff
@@ -213,10 +213,34 @@ struct cudnn_convolution_impl_base_t
     };
 
     status_t create_and_set_convolution_desc(const convolution_pd_t *pd) {
+
+        cudnnMathType_t cudnn_math_type = CUDNN_TENSOR_OP_MATH;
+
+        if (computation_data_type == CUDNN_DATA_FLOAT) {
+
+            dnnl_fpmath_mode_t dnnl_fpmath_mode;
+            dnnl_get_default_fpmath_mode(&dnnl_fpmath_mode);
```

Comment:
the default `fpmath_mode`  can be overwritten with primitive attributes.
There are two options here:
- you can use swtich as below on `pd->attr()->fpmath_mode_`
- you can use the `pd->attr()->mayidownconvert()` function as defined [here](https://github.com/oneapi-src/oneDNN/blob/b7fd832ace0d893fe44ea3688de75ff04747eae0/src/common/primitive_attr.hpp#L677). This will do some additional checks and would align on the semantics of the rest of the lib.

Reply:
Thanks for the information, @mgouicem.

I implemented the first option.